src/test/java/com/thealgorithms/datastructures/hashmap/hashing/MajorityElementTest.java

```diff
@@ -0,0 +1,48 @@
+package com.thealgorithms.datastructures.hashmap.hashing;
+
+import com.thealgorithms.datastructures.hashmap.hashing.MajorityElement;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import java.util.ArrayList;
+
+public class MajorityElementTest{
+    @Test
+    void testMajorityWithSingleMajorityElement() {
+        int[] nums = {1, 2, 3, 4, 5, 6, 7, 8, 9, 9, 9, 9};
+        List<Integer> expected = Collections.emptyList(); 
+        List<Integer> actual = MajorityElement.majority(nums);
+        assertEquals(expected, actual);
+    }
```

Comment:
The test name is `testMajorityWithSingleMajorityElement`, but then it checks that `List<Integer> expected = Collections.emptyList();`. Please rename tests to be consistent with their content

Reply:
done